framework/include/problems/DisplacedProblem.h

```diff
@@ -268,6 +269,7 @@ class DisplacedProblem : public SubProblem
   virtual void cacheJacobianNonlocal(THREAD_ID tid);
   virtual void cacheJacobianNeighbor(THREAD_ID tid) override;
   virtual void addCachedJacobian(THREAD_ID tid) override;
+  virtual void addCachedJacobianContributions(THREAD_ID tid) override;
```

Comment:
So what I've done is eliminate the `cached_jacobian_contributions*` members in `Assembly` and I've deprecated methods with "Contribution" in the name because it is very confusing to me.